resources/views/platform/components/game-list-item/primary-meta.blade.php

```diff
@@ -84,17 +84,21 @@
 
             @if ($numPossiblePoints)
                 <p>
-                    @if ($numAwardedSoftcorePoints === $numPossiblePoints || $numAwardedHardcorePoints === $numPossiblePoints)
-                        All <span class="font-bold">{{ localized_number($numPossiblePoints) }}</span>
-                    @else
-                        <span class="font-bold">{{ localized_number($numAwardedSoftcorePoints ?? $numAwardedHardcorePoints ?? 0) }}</span>
-                        of
-                        <span class="font-bold">{{ localized_number($numPossiblePoints) }}</span>
-                    @endif
-
-                    @if ($numAwardedSoftcorePoints > $numAwardedHardcorePoints) softcore @endif
+                    @php
+                        $exclusiveSoftcorePoints = max($numAwardedSoftcorePoints - $numAwardedHardcorePoints, 0);
+                        $leftPoints = $numAwardedHardcorePoints >= $exclusiveSoftcorePoints ? $numAwardedHardcorePoints : $exclusiveSoftcorePoints;
+                    @endphp
 
+                    <span class="font-bold">{{ localized_number($leftPoints) }}</span>
+                    of
+                    <span class="font-bold">{{ localized_number($numPossiblePoints) }}</span>
                     points
+
+                    @if ($exclusiveSoftcorePoints > 0 && $exclusiveSoftcorePoints < $numAwardedHardcorePoints)
+                        (+<span class="font-bold">{{ localized_number($exclusiveSoftcorePoints) }}</span> softcore)
+                    @elseif ($exclusiveSoftcorePoints > $numAwardedHardcorePoints)
```

Comment:
This needs a `$numAwardedHardcorePoints > 0`:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/316da814-746b-47b1-ade5-1571102d9795)

Reply:
Thanks, added in latest.